### ISSUE TITLE
Attendance and grades

### ISSUE THREAD

**Issue by MaxMessinger** [ErikBoesen/schoolopy#52]
Hi, me again - I would be interested in implementing grades and attendance in the module, which seems to be on some sort of todo list. Do you have any advice on how to do so? I imagine it would just be a matter of knowing the proper url for the API, no? Thanks!

**Comment by ErikBoesen** [comment]
Yes, you should be able to do this by investigating the documentation here: https://developers.schoology.com/api-documentation/rest-api-v1

**Comment by MaxMessinger** [comment]
Thanks!
